src/estrutura_sequencial/exercicio_treze.py

```diff
@@ -0,0 +1,42 @@
+"""
+Faça um Programa que calcule o excesso de peixes#13
+
+João Papo-de-Pescador, homem de bem, comprou um microcomputador para controlar
+o rendimento diário de seu trabalho. Toda vez que ele traz um peso de peixes maior
+que o estabelecido pelo regulamento de pesca do estado de São Paulo (50 quilos) deve
+pagar uma multa de R$ 4,00 por quilo excedente. João precisa que você faça um programa
+que leia a variável peso (peso de peixes) e calcule o excesso. Gravar na variável excesso
+a quantidade de quilos além do limite e na variável multa o valor da multa que João deverá
+pagar. Imprima os dados do programa com as mensagens adequadas.
+"""
+
+def calcula_excesso_peixes(peso_peixe):
+     
+    if not isinstance(peso_peixe, (float, int)):
+          raise Exception("Informe o numero correto de quilos de peixes.")
+     
+    elif peso_peixe < 0:
+          raise Exception("O peso dos peixes não pode ser negativo!!!")
+     
+    excesso = peso_peixe - 50
+    return max(0, excesso)
+
+def calcula_valor_multa(excesso):
+    return excesso * 4 if excesso > 0 else 0
```

Comment:
[dica] Aqui sabemos que o excesso é maior ou igual a zero por conta da função `calcula_excesso_peixes`, não precisaria do if...

O 4 tambem poderia estar definido como uma constante, tipo `MULTA_KG_EXCEDENTE = 4`, do jeito que está aqui é de dificil dedução e legibilidade.

```suggestion
def calcula_valor_multa(excesso):
    MULTA_KG_EXCEDENTE = 4
    return excesso * MULTA_KG_EXCEDENTE
```